LongWarOfTheChosen/Src/LW_Overhaul/Classes/LWTemplateMods.uc

```diff
@@ -2689,6 +2689,12 @@ function RewireTechTree(X2StrategyElementTemplate Template, int Difficulty)
 		if (TechTemplate.DataName == 'HeavyWeapons') // remove the alternative access to the heavy weapons proving ground project for sparks
 			TechTemplate.AlternateRequirements.Length = 0;
 
+		// Change the special requirements functions for Mechanized Warfare & SPARKs.
+		if (TechTemplate.DataName == 'MechanizedWarfare')
+			TechTemplate.Requirements.SpecialRequirementsFn = class'LWDLCHelpers'.static.IsMechanizedWarfareAvailable;
```

Comment:
Are you sure about this? Shouldn't the special requirements function be the same as for BuildSpark, i.e. has Lost Towers been completed?

Reply:
I think this is the right approach, i.e. replacing the SpecialRequirementsFn, I'm just not sure this is the right function. I'm pretty sure the point is for the SpecialRequirementsFn to override the prerequisites, which is ADVENT Robotics for Mechanized Warfare.